assembl/alembic/versions/39b04e9d2716_remove_video_module.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding=utf-8 -*-
+"""Remove_video_module
```

Comment:
as it is not just the removal of video module, you should rename the migration and change this comment